integration/raft/channel_participation_test.go

```diff
@@ -1302,6 +1309,14 @@ func submitTxn(o *nwo.Orderer, env *common.Envelope, n *nwo.Network, expectedCha
 	}, n.EventuallyTimeout).Should(Equal(expectedChannelInfo))
 }
 
+func deliverBlock(o *nwo.Orderer, n *nwo.Network, channel string) {
+	env := CreateDeliverEnvelope(n, o, 0, channel)
+	Expect(env).NotTo(BeNil())
+
+	By("fetching block from " + o.Name)
+	Eventually(deliverFunc(n, o, env), n.EventuallyTimeout, time.Second).ShouldNot(BeNil())
+}
+
```

Comment:
maybe we could reuse `FetchBlock` from the same package? https://github.com/hyperledger/fabric/blob/25457a3ce5b2b9fa5e6908236cefbbb7cbb66670/integration/raft/client.go#L19

Reply:
Ah, thanks! Somehow kept overlooking that that function existed. Done.